docs/built-in-functions.rst

```diff
@@ -0,0 +1,237 @@
+.. index:: function, built-in;
+
+.. _built_in_functions:
+
+***********************
+Built in Functions
+***********************
+
+Viper contains a set amount of built in functions that would be timely and/or unachievable to write in Viper.
+
+.. _functions:
+
+Functions
+=========
+The first list dictates each argument and the second sublist lists the types allowed as input.
```

Comment:
Maybe consider some simple examples.

Reply:
In fact linking to the online execution environment (compiler / remix etc...) with executable examples in context would be super useful to developers but that would require writing those examples so should be a separate task

Reply:
I'm a fan of simple examples, down the road maybe we could have a section in `examples/` dedicated to showing off specific viper functionality?